lib/server.js

```diff
@@ -794,6 +796,10 @@ WalletService.prototype.createTx = function(opts, cb) {
   if (!Utils.checkRequired(opts, ['toAddress', 'amount', 'proposalSignature']))
     return cb(new ClientError('Required argument missing'));
 
+  var feePerKb = opts.feePerKb || 10000;
+  if (feePerKb > 10000)
```

Comment:
Personal experience, I had 3 "free" transactions in the past 3 months. The first confirmation has been taking longer and longer... this last time it took me 5 blocks to get in there.

As there is a limited amount of space per block allotted for "free" transactions, This would not be friendly to newbies...

However, I am not against the idea of allowing free transactions... but since most wallets don't allow setting fees below a certain amount... I think it would be wise to follow suit.